CMakeLists.txt

```diff
@@ -87,6 +87,11 @@ elseif(CMAKE_C_COMPILER_ID STREQUAL "Intel")
 endif(CMAKE_C_COMPILER_ID STREQUAL "GNU")
 
 #-----------------------------------------------------------------------
+# Check for prerequisite libraries
+
+find_package(Threads)
```

Comment:
I assume there was a conflict with the `ENABLE_` flags and the threads libraries.  Cmake chooses differently?